### PR TITLE
Rename Factory Reset option to "Keep cached kubernetes images"

### PR DESCRIPTION
**Pull request by jandubois**
because "Container System Images" is kind of abstract? What are they, and why would you or wouldn't you want to keep them?

See also https://github.com/rancher-sandbox/rancher-desktop/issues/2494#issuecomment-1189536627

### CODE REVIEW

**Review by ericpromislow — CHANGES_REQUESTED** [review]
I assume you want to change the wording in https://github.com/rancher-sandbox/rancher-desktop/blob/658b62e498c2b450a33148f5e14bb1c8df7d2325/src/go/rdctl/cmd/factoryReset.go#L37 to be consistent with this.

Note that in the UI the default is to remove the cached images, and on the command-line the default is to keep them.

**Comment by jandubois** [comment]
> I assume you want to change the wording in
> 
> https://github.com/rancher-sandbox/rancher-desktop/blob/658b62e498c2b450a33148f5e14bb1c8df7d2325/src/go/rdctl/cmd/factoryReset.go#L37
> 
> to be consistent with this.
> Note that in the UI the default is to remove the cached images, and on the command-line the default is to keep them.

Yeah, I don't know. The fact that the defaults are not the same already seems confusing, so I decided to leave it alone for now. I still expect to rename a lot of `rdctl` commands and options when we go to the `v1` API and make it non-experimental.

But yeah, I guess I'll change "system images" to "cached kubernetes images" everywhere, for some level of consistency.

**Comment by jandubois** [comment]
And it is getting worse by `wsl-helper factory-reset` having a `--keep-system-images` option, while `rdctl factory-reset` has `--remove-kubernetes-cache`. 🙍 

At least `wsl-helper` should not be exposed to the user, so I'll leave this alone for now.

File: src/pages/Troubleshooting.vue [hunk 1]
```diff
@@ -57,7 +57,7 @@
             data-test="factoryResetButton"
             label="Factory Reset"
             value="auto"
-            :options="[{id: 'keepImages', label: 'Keep Container System Images'}]"
+            :options="[{id: 'keepImages', label: 'Keep cached kubernetes images'}]"
```
Comment (ericpromislow) [inline src/pages/Troubleshooting.vue:60]: `Kubernetes` should be capitalized, but I'm fine not capitalizing the other two words.

**Review by ericpromislow — APPROVED** [review]

**Pull request merged**